src/main/groovy/de/triplet/gradle/play/PlayPublishApkTask.groovy

```diff
@@ -45,12 +46,19 @@ class PlayPublishApkTask extends PlayPublishTask {
         if (extension.untrackOld && !"alpha".equals(extension.track)) {
             def untrackChannels = "beta".equals(extension.track) ? ["alpha"] : ["alpha", "beta"]
             untrackChannels.each { channel ->
-                Track track = edits.tracks().get(variant.applicationId, editId, channel).execute()
-                track.setVersionCodes(track.getVersionCodes().findAll {
-                    it > apk.getVersionCode()
-                });
-
-                edits.tracks().update(variant.applicationId, editId, channel, track).execute()
```

Comment:
If possible I'd rather check whether `versionCodes` is empty instead of running into an exception that could have been avoided.